esp-wifi/src/ble/os_adapter_esp32c2.rs

```diff
@@ -118,16 +118,16 @@ pub(super) unsafe extern "C" fn esp_intr_alloc(
 }
 
 pub(super) fn ble_rtc_clk_init() {
-    // stealing RADIO_CLK is safe since it is passed (as mutable reference or by
+    // stealing BT is safe since it is passed (as mutable reference or by
     // value) into `init`
-    let radio_clocks = unsafe { RADIO_CLK::steal() };
-    RadioClockController::new(radio_clocks).ble_rtc_clk_init();
+    let mut bt = unsafe { BT::steal() };
+    bt.ble_rtc_clk_init();
 }
 
 pub(super) unsafe extern "C" fn esp_reset_rpa_moudle() {
     trace!("esp_reset_rpa_moudle");
-    // stealing RADIO_CLK is safe since it is passed (as mutable reference or by
+    // stealing BT is safe since it is passed (as mutable reference or by
```

Comment:
These comments no longer needs the parens, as peripherals are now always passed by value